k9mail/src/main/java/com/fsck/k9/activity/MessageLoaderHelper.java

```diff
@@ -416,7 +421,7 @@ public void loadMessageRemoteFailed(Account account, String folder, String uid,
         void onMessageDataLoadFailed();
 
         void onMessageViewInfoLoadFinished(LocalMessage localMessage, MessageViewInfo messageViewInfo);
-        void onMessageViewInfoLoadFailed(LocalMessage localMessage);
+        void onMessageViewInfoLoadFailed(LocalMessage localMessage, MessageViewInfo messageViewInfo);
```

Comment:
I suppose. I only left it for consistency of a message view info load returning callback in `MessageLoaderCallbacks`